src/components/Contact/Contact.tsx

```diff
@@ -25,9 +25,7 @@ const Contact: FC = () => {
                 </FancyLink>
 
                 <span>🙋🏼‍♂️</span>
-                <FancyLink to="https://bradgarropy.com/ama">
-                    ask me anything
-                </FancyLink>
+                <FancyLink to="/ama">ask me anything</FancyLink>
```

Comment:
Making this a relative link means this will be a client side route, which avoids running the `root` loader again. Not a big deal, but a slight optimization.